docs/install-docker.md

```diff
@@ -52,7 +55,41 @@ sudo systemctl start docker
 sudo usermod -aG docker $(whoami)
 ```
 
-Note that CanDIG requires **Docker Compose v2**, which is provided alongside the latest version of Docker. Versions of Docker which do not provide Docker Compose will unfortunately not work with CanDIG.
```

Comment:
Is this note no longer relevant?

Reply:
It's still there, I moved it to the top section where it makes more sense.

Reply:
oh right sorry I missed that!